docs/virtualkeyboard/index.html

```diff
@@ -128,7 +142,7 @@ <h2>The <dfn>VirtualKeyboard</dfn> Interface</h2>
                       Call the system API to show the VK.
                   </li>
                   <li>
-                    When the VK is shown by the system, fire {{VirtualKeyboard/ongeometrychange()}} event.
+                    When the VK is shown by the system, <a href="https://html.spec.whatwg.org/#in-parallel">in parallel</a> the OS reports the bounding rectangle of the keyboard. Update the {{VirtualKeyboard/boundingRect}} which is the intersection of the keyboard rectangle reported by the OS and the layout viewport. Fire {{VirtualKeyboard/ongeometrychange()}} event to JS.
```

Comment:
You can move the "in parallel" bit up, into something like:
"""
<li>[=In parallel=], follow these steps:
* Wait for the virtual keyboard to be shown by the system.
* Call "set the virtual keyboard bounding rect" with the keyboard's OS reported bounding rectangle and the [=document=]'s [=viewport=].
* [=Fire an event=] named "geometrychange" at [=this=].

Reply:
Done. Couldn't use [=viewport=] because for some reason css xref's are not working?

Reply:
@marcoscaceres - can you help on the [[=viewport=]](https://drafts.csswg.org/css2/#viewport) front?